incorrect error message for violations of lowerCamelCase

### Describe the bug

The error message produced when a message key only violates the stipulation of lowerCamelCase is wrong, it talks about special characters.  Following is an example.

```
/Users/mspreitz/go/src/github.com/MikeSpreitzer/logcheck-test/main.go:15:2: Key positional arguments "Fubar" are expected to no special characters. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.
```


### Steps To Reproduce

Apply logcheck to any file with a key that starts with a capital letter.

### Expected Behaviour

I expected the error message to say that the violation is about lowerCamelCase rather than special characters.

--- a/logcheck/pkg/logcheck.go
+++ b/logcheck/pkg/logcheck.go
@@ -660,7 +660,7 @@ func keysCheck(keyValues []ast.Expr, fun ast.Expr, pass *analysis.Pass, funName 
 		if !match {
 			pass.Report(analysis.Diagnostic{
 				Pos:     fun.Pos(),
-				Message: fmt.Sprintf("Key positional arguments %s are expected to no special characters. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.", lit.Value),
+				Message: fmt.Sprintf("Key positional arguments %s are expected to be alphanumeric and start with either one lowercase or two uppercase letters. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.", lit.Value),
 			})
 		}
 	}

